jobs/eirini-extensions/templates/bpm.yml.erb

```diff
@@ -8,3 +8,14 @@ processes:
       OPERATOR_WEBHOOK_HOST: "<%= p("eirini-extensions.operator_webhook_host") %>"
       OPERATOR_WEBHOOK_PORT: "<%= p("eirini-extensions.operator_webhook_port") %>"
       NAMESPACE: "<%= p("eirini-extensions.namespace") %>"
+    <% if properties.opi&.k8s&.host_url.nil? %>
```

Comment:
what's with the `&`?

Reply:
can we have a property `properties.eirini.in_cluster`, defaulted to `false`?
then this can be set if `properties.eirini.in_cluster` is set.
VM installations would keep it to `false`, we would set it to `true` from SCF opinions.yaml

Reply:
`&` -> http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/
We can have the new property though it would do essentially the same thing for now. If one sets `host_url` then we are trying to talk to an external kube. Would we ever want eirini from kube cluster#1 to deploy to kube cluster#2? If that's the case then the current code is more useful I think.